plugins/obs-qsv11/obs-qsv11.c

```diff
@@ -772,8 +772,6 @@ static void update_params(struct obs_qsv *obsqsv, obs_data_t *settings)
 	     "\twidth:          %d\n"
 	     "\theight:         %d",
 	     voi->fps_num, voi->fps_den, width, height);
-
-	info("debug info:");
```

Comment:
I think its intended really to catch the Initialization errors and head them... either way if we think its worth separating these elements lets move the message output before those are printed instead of here.

Reply:
Since the latter are not user-specified settings, I'd prefer to keep them separate for now. Either keep this line, or move it to a more appropriate location.

Reply:
sure, ill drop the commit and bring it up another time.